src/summary-table.js

```diff
@@ -142,19 +142,20 @@ export function SummaryTable({
                     role="cell"
                     key={idx}
                     rowSpan={trueOnly ? 1 : row["cycleRowspan"] || 1}
+                    className={idx}
```

Comment:
Is there a reason for adding this class here and below? If not, I think it makes sense to remove it, so we don't get confused later.